Name: Haptic Feedback Manager
Version: 1.3.5-1
Author: Fritos1406
Description: Enables Vibrate to any Tap on an object/icon..etc.

--- .orig/usr/palm/frameworks/mojo/builtins/palmInitFramework200_72.js
+++ /usr/palm/frameworks/mojo/builtins/palmInitFramework200_72.js
@@ -10795,6 +10795,10 @@
 
 
 sendTap:function(triggeringEvent){
+		this.serviceRequest = new Mojo.Service.Request					
+		("palm://com.palm.vibrate", {
+		method: 'vibrate', parameters: { 'period': 0,'duration': 75 }
+		});
 var tapEvent,focusedElement;
 focusedElement=Mojo.View.getFocusedElement(this.downTarget.ownerDocument.body);
 if(focusedElement){
@@ -11777,6 +11781,10 @@
 
 
 tapEvent:function(event){
+		this.serviceRequest = new Mojo.Service.Request					
+		("palm://com.palm.vibrate", {
+		method: 'vibrate', parameters: { 'period': 0,'duration': 75 }
+		});
 this.dragEnd(event);
 },
 
